graphql_query_derive/src/shared.rs

```diff
@@ -110,8 +112,15 @@ pub(crate) fn response_fields_for_selection(
 
                 let schema_field = &schema_fields
                     .iter()
-                    .find(|field| field.name.as_str() == name.as_str())
-                    .ok_or_else(|| format_err!("Could not find field: {}", name.as_str()))?;
+                    .find(|field| &field.name == name)
+                    .ok_or_else(|| {
+                        format_err!(
+                            "Could not find field `{}` on `{}`. Available fields: `{}`.",
+                            name.as_str(),
+                            type_name,
+                            schema_fields.iter().map(|ref field| &field.name).format("`, `"),
```

Comment:
the `format` method is really neat!

Reply:
Yeah, it makes things like this really easy.